release/mk-release-snapshots.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/sh
+#
+# Publish Alpha releases (a.k.a. daily snapshot)
```

Comment:
To match the terminology used in README.md:

```suggestion
# Publish the latest development release (a.k.a. daily snapshot)
```

If you do want to use an additional "alpha" term, we still can do that without introducing a mismatch:

```suggestion
# Publish the latest development release (a.k.a. daily snapshot or alpha release)
```